src/Container.php

```diff
@@ -285,7 +285,7 @@ public static function create(): self
                 $container->getPrinter(),
                 $container->getMutantCodeFactory(),
             ),
-            Differ::class => static fn (): Differ => new Differ(new BaseDiffer(new UnifiedDiffOutputBuilder())),
+            Differ::class => static fn (): Differ => new Differ(new BaseDiffer(new UnifiedDiffOutputBuilder(''))),
```

Comment:
I hope you are ok removing the `"--- Original\n+++ New\n"` header, which only adds visual clutter to the output